s4e-web/src/app/views/map-view/zk/report-modal/report-modal.component.ts

```diff
@@ -67,14 +68,21 @@ export class ReportModalComponent extends FormModalComponent<'report'> {
 
   accept() {
     if (this.form.invalid) {
-      console.log(this.form.errors);
+      console.error(this.form.errors);
```

Comment:
log (or debug for that metter) is correct. Either remove logging at all, or leave log. `console.error` is reserved for things which went wrong. Here we expect those errors and handle them properly.

Some audit tools (like chrome audits) really do not like when you throw errors into the console